scripts/postgres_setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
This all feels a bit hokey, but (after some searching) I found there is [no mainline way](https://github.com/docker-library/postgres/issues/191) out-of-the-box to set up a simple "run this script at startup" process for postgres' docker image so I created a custom entrypoint/command here to accomplish the task of setting up DBs and doing other DB work to orchestrate data as needed.